packages/FixedAssetDepreciation/src/Services/DepreciationManager.php

```
@@ -0,0 +1,242 @@
+
+        return $this->revaluationService->revalue(
+            $assetId,
+            $newValue,
+            $newSalvageValue,
+            $type,
+            $reason,
+            $glAccountId
+        );
+    }
+
+    public function calculateTaxDepreciation(string $assetId, string $taxMethod, int $taxYear): DepreciationAmount
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$taxMethod'. (undefined)

(UnusedFormalParameter)